File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.java
class Solution {
    public boolean checkIfPangram(String sentence) {
        
        if (sentence.length() < 26) {
            return false;
        }
        
        HashSet<Character> set = new HashSet<>();
        
        for (int i=0; i<sentence.length(); i++) {
            set.add(sentence.charAt(i));
        }
        
        if (set.size() != 26) {
            return false;
        }
        
        return true;
    }
}

File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.py
class Solution(object):
    def checkIfPangram(self, sentence):
        """
        :type sentence: str
        :rtype: bool
        """
        
        if len(sentence) < 26:
            return False
        
        ahplabets = set()
        
        for char in sentence:
            ahplabets.add(char)
            
        if len(ahplabets) == 26:
            return True
        return False
            
        